public/index.html

```diff
@@ -19,7 +42,13 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>React App</title>
+    <title>mooniak.com</title>
+    <style media="screen">
+    @font-face {
```

Comment:
Import fonts in a separate CSS file like in here: https://github.com/mooniak/fontpad